include/mpd/database.h

```diff
@@ -80,14 +82,14 @@ mpd_send_list_all_meta(struct mpd_connection *connection, const char *path);
  * To read the response, you may use mpd_recv_entity().
  *
  * @param connection the connection to MPD
- * @param path the directory to be listed
+ * @param path an optional the directory to be listed
```

Comment:
Typo `the`?

Reply:
yup, i've added a new commit to fix this